scripts/influxd-systemd-start.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash -e
+
+/usr/bin/influxd &
```

Comment:
Rather than doing this background-process dance with the PID file, you could use [`ExecStartPost=`](https://www.freedesktop.org/software/systemd/man/systemd.service.html#ExecStartPre=) in the unit file to call this script separately. This would also have the advantage of allowing the system administrator to override it with a dropin file if the script causes a problem like in https://github.com/influxdata/influxdb/issues/21967